examples/ex9.cpp

```diff
@@ -127,6 +134,12 @@ int main(int argc, char *argv[])
                   "Use binary (Sidre) or ascii format for VisIt data files.");
    args.AddOption(&vis_steps, "-vs", "--visualization-steps",
                   "Visualize every n-th timestep.");
+   args.AddOption((int*) &fem_type, "-fem", "--fem_type",
+                  "Choose fem discretization (CG or DG).");
+   args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
+                  "--no-partial-assembly", "Enable Partial Assembly.");
```

Comment:
Maybe change the help string to indicate that DG doesn't work with PA, e.g.:

```suggestion
                  "--no-partial-assembly", "Enable Partial Assembly (CG only).");
```